pkg/codegen/convert-gnmi-oapi/typedef.tmpl

```diff
@@ -10,15 +10,47 @@ type ModelPluginDevice struct {
 {{range .Types}}{{$elevenchars := .TypeName}}{{$ltn := len .TypeName}}{{if lt 11 $ltn}}{{$elevenchars = slice .TypeName 0 11}}{{end}}
 {{if printf "RequestBody" | eq $elevenchars}}//Ignoring {{.TypeName}}
 {{else}}
-// to{{.TypeName}} converts gNMI to OAPI.{{$tn := .TypeName}}
+// to{{.TypeName}} converts gNMI to OAPI from the top level device.{{$tn := .TypeName}}
 func (d *ModelPluginDevice) to{{.TypeName}}(params ...string) (*types.{{.TypeName}}, error) {
```

Comment:
a thing that I noticed yesterday while playing around with other thing is that all these methods are private so they can't be imported in other packages. Do we think it's worthy exposing them?